test/sql/function/numeric/test_bit_count.test

```diff
@@ -6,15 +6,15 @@ statement ok
 PRAGMA enable_verification
 
 statement ok
-CREATE TABLE bits(t tinyint, s smallint, i integer, b bigint)
+CREATE TABLE bits(t tinyint, s smallint, i integer, b bigint, h hugeint)
 
 statement ok
-INSERT INTO bits VALUES (NULL, NULL, NULL, NULL), (31, 1023, 11834119, 50827156903621017), (-59, -517, -575693, -9876543210)
+INSERT INTO bits VALUES (NULL, NULL, NULL, NULL, NULL), (31, 1023, 11834119, 50827156903621017, 3141592653589793238462643383279528841), (-59, -517, -575693, -9876543210, -148873535527910577765226390751398592512)
```

Comment:
I was actually just amused that you had these large numbers with so few bits set. But it occurs to me now that it would be good to have one high order bit set . The negative is good for the high order part, but just check that the low order high bit is included too?

Reply:
I *think* it is working right - based upon how it compares to tiny int:
```
D select bit_count(170141183460469231731687303715884105727), bit_count(-170141183460469231731687303715884105727);
┌────────────────────────────────────────────────────┬─────────────────────────────────────────────────────┐
│ bit_count(170141183460469231731687303715884105727) │ bit_count(-170141183460469231731687303715884105727) │
├────────────────────────────────────────────────────┼─────────────────────────────────────────────────────┤
│ 127                                                │ 2                                                   │
└────────────────────────────────────────────────────┴─────────────────────────────────────────────────────┘
D select bit_count(cast(127 as tinyint)), bit_count(cast(-127 as tinyint));
┌─────────────────────────────────┬──────────────────────────────────┐
│ bit_count(CAST(127 AS TINYINT)) │ bit_count(CAST(-127 AS TINYINT)) │
├─────────────────────────────────┼──────────────────────────────────┤
│ 7                               │ 2                                │
└─────────────────────────────────┴──────────────────────────────────┘
```

Does that seem ok?

Reply:
Seems ok :)